backend/src/app/routers/skills/skills.py

```diff
@@ -6,27 +6,27 @@
 from src.app.logic import skills as logic_skills
 from src.app.routers.tags import Tags
 from src.app.schemas.skills import SkillBase, Skill, SkillList
-from src.app.utils.dependencies import require_auth
+from src.app.utils.dependencies import require_admin
 from src.database.database import get_session
 
 skills_router = APIRouter(prefix="/skills", tags=[Tags.SKILLS])
 
 
-@skills_router.get("", response_model=SkillList, tags=[Tags.SKILLS], dependencies=[Depends(require_auth)])
+@skills_router.get("", response_model=SkillList, tags=[Tags.SKILLS], dependencies=[Depends(require_admin)])
```

Comment:
I think the frontend students page uses this request to get all available skills to filter on, so this will make it crash unfortunately. 

```suggestion
@skills_router.get("", response_model=SkillList, tags=[Tags.SKILLS], dependencies=[Depends(require_coach)])
```